libraries/core/src/main/java/com/paritytrading/philadelphia/FIXConnection.java

```diff
@@ -386,8 +395,11 @@ public void keepAlive() throws IOException {
      * @throws IOException if an I/O error occurs
      */
     @Override
+    @SuppressWarnings("EmptyTryBlock")
```

Comment:
I think this is IntelliJ IDEA'ism? I would rather leave it out. 🙂

Reply:
Getting rid of the ones I added. There is one more in `FIXConfigTest` (and yes, IntelliJ adds these, but it is a standard Java mechanism to suppress a warning. I.e. `javac` also uses it.)